### DISCUSSION TITLE
Docusaurus-v3: How do you customize the results display?

### DISCUSSION THREAD

**Discussion by osseonews** [oramasearch/orama#724] [Q&A]
How do you customize how the results are displayed with the docusaurus-v3 plugin? On the original docusaurus plugin you had folders in the theme like "SearchResults" which had the code for displaying the result, so this was easy to customize. But in the V3 plugin all you have is the Search bar and I can't for the life of me figure out where the results are generated in the code and how to customize these results. I don't even see in the plugin anything related to the results. How is this done, so we can customize it? Thanks.

**Comment by arbcar** [comment]
Any idea ? Thanks

**Comment by micheleriva** [comment]
@allevo, @matijagaspar, @raiindev I may need your help here 🙏

**Comment by AlexandraKosa** [comment]
Hi, I need to know it as well. Thanks.
